### PR TITLE
#356: [TextOverflow] Failed to execute 'getComputedStyle' on 'Window': parameter 1 is not of type 'Element'. (closes #356)

### LINKED ISSUE #356: [TextOverflow] Failed to execute 'getComputedStyle' on 'Window': parameter 1 is not of type 'Element'.
## description

under certain circumstances, TextOverflow is trying to get the computed style of a `null` `React.findDOMNode` return value:
```
Failed to execute 'getComputedStyle' on 'Window': parameter 1 is not of type 'Element'.
```

## specs

for the time being, just patch it and hide the possibly-null access behind a check

### PR DESCRIPTION
**Pull request by giogonzo**
Issue #356

### CODE REVIEW

File: src/text-overflow/TextOverflow.js [hunk 1]
```diff
@@ -41,10 +41,12 @@ export default class TextOverflow extends React.Component {
   logWarnings = () => {
     warn(() => {
       const node = React.findDOMNode(this.refs.text).parentNode.parentNode;
```
Comment (FrancescoCioria) [inline src/text-overflow/TextOverflow.js]: what if `React.findDOMNode(this.refs.text)` is `null`? Are we sure this could never happen?

Comment (giogonzo) [inline src/text-overflow/TextOverflow.js]: sure, didn't notice the `.parent..`, fixing now

**Comment by giogonzo** [comment]
@FrancescoCioria should be "ok" now

**Comment by FrancescoCioria** [comment]
:+1:

**Pull request merged**